### PR TITLE
Mechanical integration README.md

### PR DESCRIPTION
**Pull request by dcschooley**

### CODE REVIEW

**Review by gfosmire — CHANGES_REQUESTED** [review]
Broken image link in the Mechanical Integration README.md, should change the image filename to be lowercase to be inline with other filenames.

Currently is
images/mechanical_integration/Differential_Pivot.png
should be
images/mechanical_integration/differential_pivot.png

**Comment by apollokit** [comment]
We really ought to updating these readmes in separate branches (i see at least mechanical innovation and electrical calibration within this one). Now that it's done though, might as well just continue forward

**Comment by dcschooley** [comment]
The file name should be fixed now.

**Comment by apollokit** [comment]
I force pushed to remove the patch and electrical build related commits. They shouldn't have been here, they're from this PR https://github.com/nasa-jpl/open-source-rover/pull/243/ and this PR https://github.com/nasa-jpl/open-source-rover/pull/245

File: mechanical/mechanical_integration/README.md [hunk 1]
```diff
@@ -1,32 +1,154 @@
...
-| --------------            | -------            |
-| Cost without 3D printer   | 56.58              |
-| Cost with 3D printer      | 56.58              |
-| Date Added                | 4/11               |
-| Group/Organization/School | JPL                |
-| Time to build             | 2 hrs              |
+**Figure 1: All subassemblies completed**
 
-## Skills
+## 1.1 Mounting Wheels
```
Comment (apollokit) [inline mechanical/mechanical_integration/README.md:23]: Should make this table two columns. Refer to https://github.com/nasa-jpl/open-source-rover/blob/master/examples/README_example.md

File: mechanical/mechanical_integration/README.md [hunk 2]
```diff
@@ -1,32 +1,154 @@
...
 
-## Tools
 
-### Mandatory 
-  * Allen Key set
-  * Imperial Wrench Set
+  
 
-### Optional
+1. Mounting the Middle Wheel: Mount one base wheel (not the corner wheels) to the lowest arm of each of the two rocker bogie subassemblies using a 1.5 inch channel **S1** and channel connectors S6. See Figures 2 and 3. Note that you should mirror the wheels so that they are both on the ”outside” of the rocker bogey subassemblies.
```
Comment (apollokit) [inline mechanical/mechanical_integration/README.md:41]: also would be good to follow the N.N.N numbering scheme from https://github.com/nasa-jpl/open-source-rover/blob/master/examples/README_example.md for these sub-subsections

helps to keep the reader oriented for where they are in the file

**Review by apollokit — CHANGES_REQUESTED** [review]
Made some requests related to the document structure and one table.

**Comment by dcschooley** [comment]
Please go ahead and review this one.

File: mechanical/mechanical_integration/README.md [hunk 3]
```diff
@@ -1,32 +1,166 @@
...
+<tr>
+	<td><img src="../../images/mechanical_integration/diff_step_6b.png"></td>
+	<td><img src="../../images/mechanical_integration/diff_step_7.png"></td>
+</tr>
+</table>
+
+**Figure 8: Inserting differential pivot joint**
+
+   
+###1.5.2 Attaching the differential pivot arms
```
Comment (Achllle) [inline mechanical/mechanical_integration/README.md:139]:
```suggestion
### 1.5.2 Attaching the differential pivot arms
```
It doesn't render well without the space.
You can just hit 'commit' to include this proposed suggestion so you don't actually have to go back and commit manually

Comment (dcschooley) [reply mechanical/mechanical_integration/README.md]: done

File: mechanical/mechanical_integration/README.md (hunk 2 above)
Comment (dcschooley) [reply mechanical/mechanical_integration/README.md]: done

File: mechanical/mechanical_integration/README.md [hunk 4]
```diff
@@ -1,32 +1,154 @@
-# Mechanical Integration
-This is the final mechanical integration of all of the other subassemblies to create the final rover.
 
-<img src="/images/mechanical_integration/final_title.png" width="80%">
 
-## Features
-  * All subassemblies can be combined together with the addition of very few additional components.
+#  1 Mechanical/Structural Assembly
```
[the remark this answers is not in the document]
Comment (dcschooley) [reply mechanical/mechanical_integration/README.md]: done

**Review by apollokit — APPROVED** [review]

**Review by Achllle — APPROVED** [review]
lgtm

**Pull request merged**